lib/eslint/flat-eslint.js

```diff
@@ -1212,11 +1212,31 @@ class FlatESLint {
     }
 }
 
+/**
+ * Returns whether flat config should be used.
+ * @returns {Promise<boolean>} Where flat config should be used.
```

Comment:
Not your fault, but might as well fix it now.

```suggestion
 * @returns {Promise<boolean>} Whether flat config should be used.
```

Reply:
Ah I hadn't even noticed that :)

done